libs/community/langchain_community/document_loaders/confluence.py

```diff
@@ -575,10 +592,19 @@ def process_page(
             ]
             text = text + "".join(comment_texts)
 
+        if include_labels:
+            labels = [
+                label["name"]
+                for label in page.get("metadata", {})
+                .get("labels", {})
+                .get("results", [])
+            ]
```

Comment:
label would be sth like this: `{'prefix': 'global', 'name': 'database', 'id': '111111111'}` 

![Screenshot 2024-11-21 at 22 23 03](https://github.com/user-attachments/assets/9e3fca9c-10a1-4858-af16-1b12c5d3c268)

[ref](https://developer.atlassian.com/cloud/confluence/rest/v1/api-group-content/#api-wiki-rest-api-content-search-get)